jwk/interface.go

```diff
@@ -41,7 +42,8 @@ const (
 // Set is a convenience struct to allow generating and parsing
 // JWK sets as opposed to single JWKs
 type Set struct {
-	Keys []Key `json:"keys"`
+	Keys    []Key     `json:"keys"`
+	Expires time.Time `json:"expires"`
```

Comment:
RFC7515 makes no mention of this expiration, so probably should not be serialized to/from JSON.

Reply:
As far as I could tell, this `Set` struct is never used for marshal/unmarshal operations, but rather the `proxy` struct at https://github.com/lestrrat-go/jwx/blob/2bc66abc0137afe334b59779962b1fab258c630f/jwk/jwk.go#L291

Reply:
Indeed. So then it should lose its json annotations entirely.